src/Controls/src/Core/Compatibility/Handlers/Shell/iOS/ShellSearchResultsRenderer.cs

```diff
@@ -88,7 +88,14 @@ public override UITableViewCell GetCell(UITableView tableView, NSIndexPath index
 			var template = SearchHandler.ItemTemplate;
 
 			if (template == null)
+			{
+				if (!RuntimeFeature.IsShellSearchResultsRendererDefaultTemplateSupported)
+				{
+					throw new ArgumentNullException(nameof(SearchHandler.ItemTemplate), "ItemTemplate must be set on SearchHandler");
+				}
```

Comment:
It looks like we should make the feature flag around `DisplayMemberName` and the default template can still work.

TODO: how many `DisplayMemberName`-like features exist out there? Telerik, third part, etc.?